### DISCUSSION TITLE
LlavaConfig, preprocessor.json, weights .. Would someone be so kind to help me to understand how to get started?

### DISCUSSION THREAD

**Discussion by Ben-Benser** [haotian-liu/LLaVA#1063] [Q&A]
Hello Ladies and Gentlemen,

I recently started to get into AI, DNN, LLMs etc. Started to learn math by myself, courses by Andrew Ng etc.
But all this differs pretty much from how to actually get started.

So since 2 days im trying to get my locally downloaded 34b LLaVA 1.6 model initialized. Worked through HF docs as well as docs here on GitHub and other docs regarding earlier LLaVA - versions but it is quite overwhelming. Tbh I can't get it done. At first it always told me that files like preprocessor_config.json are missing as well as config-files in general. 

Got those files by searching in other llava-files, which lead to other errors. 
At the end I feel like im loosing too much time, thus came here to ask for some support. Maybe also some dedicated resources to learn, so I don't have to ask for such basics, since im sure I can't get it done because im a noob.

Would be glad if someone might response but totally get it, if no one finds the time and patience to do so.
Cheers

**Comment by n2nco** [comment]
U still kickin?
